core/src/main/scala/org/broadinstitute/dsp/workbench/welder/GoogleStorageInterp.scala

```diff
@@ -55,7 +57,7 @@ class GoogleStorageInterp(config: GoogleStorageAlgConfig, googleStorageService:
       _ <- (Stream
         .emits(blob.getContent())
         .covary[IO]
-        .through(io.file.writeAll[IO](localAbsolutePath, linerBacker.blockingContext))) ++ Stream.eval(IO.unit)
+        .through(io.file.writeAll[IO](localAbsolutePath, linerBacker.blockingContext, List(StandardOpenOption.TRUNCATE_EXISTING)))) ++ Stream.eval(IO.unit)
```

Comment:
Will this still create the file if it doesn't already exist, or do you need to include CREATE as well?